lib/helpers/versions.js

```diff
@@ -96,3 +107,26 @@ function isPastLatest(dep, version) {
   logger.warn(`No dist-tags.latest for ${dep.name}`);
   return false;
 }
+
+function computeRangedVersion(currentVersion, newVersion) {
+  const parsedRange = semverUtils.parseRange(currentVersion);
+  let rangedVersion = '';
+
+  // Check whether the current version is pinned, and if so, just return the
+  // new version, which, itself, should be a pinned version number.
+  if (parsedRange.length === 1 && parsedRange[0].operator === undefined) {
+    rangedVersion = newVersion;
```

Comment:
Simpler logic if we `return newVersion` here and the subsequent `else if` becomes an `if` ?